testkit/src/builder.rs

```diff
@@ -185,7 +215,25 @@ impl TestKitBuilder {
             .test_network
             .unwrap_or_else(|| TestNetwork::with_our_role(our_validator_id, 1));
         let genesis = network.genesis_config();
-        TestKit::assemble(TemporaryDB::new(), self.service_instances, network, genesis)
+
+        let mut runtimes = vec![];
+        if !self.rust_services.is_empty() {
```

Comment:
If I understand correctly, it's impossible to use any runtime without Rust one because Supervisor is absolute need. Why it is possible to create a testkit without Rust runtime enabled then?

Reply:
According to our discussion going to bring back the functionality of always having default Rust runtime.